core/src/main/java/Scenes/Play.java

```diff
@@ -33,6 +37,20 @@ public Play(FilePlayMain mainGame) {
     createButtons();
     addAllListeners();
     addAllActors();
+    // Creates a Music instance of the song to be used during game play.
+    /* Credit for the composition goes to Ludwig van Beethoven.
+       Credit for the performance goes to Simone Rezi.
+       Credit for hosting the file goes to musopen.org.
+     */
```

Comment:
Does this need to be right here, and not somewhere else?  If it absolutely has to be here, then either fix multiline comment formatting, or change to single-line (implementation) comment formatting (preferred).